src/Lykke.Service.PayInternal.Services/ConfirmationsService.cs

```diff
@@ -35,7 +35,7 @@ public class ConfirmationsService : IConfirmationsService
                 .WaitAndRetryAsync(
                     _retryPolicySettings.DefaultAttempts,
                     attempt => TimeSpan.FromSeconds(Math.Pow(2, attempt)),
-                    (ex, timespan) => _log.Error("Publish invoice confirmation with retry", ex));
+                    (ex, timespan) => _log.WriteError("Publish invoice confirmation with retry", null, ex));
```

Comment:
WriteError is obsolet: https://github.com/LykkeCity/Lykke.Logs/blob/master/README.md